docs/quick_start.md

```diff
@@ -115,28 +121,62 @@ In addition to several [standard Web APIs](./api/index.md#web-platform-apis), `j
   Smart functions can invoke other smart functions using `fetch`, similiar to network requests in JavaScript.
   Additionally, new smart functions can be deployed by a smart function using the [`SmartFunction`](./api/smart_function.md) API.
 
-## 2) Deploying your Smart Function
+## 2. Deploying your Smart Function
 
 First we must compile our TypeScript code to JavaScript using:
 
 ```sh
+npm install
 npm run build
 ```
 
 Once built, we can deploy the smart function to the local sandbox:
 
 ```sh
-jstz sandbox start -d
+jstz sandbox start
 jstz deploy dist/index.js
```

Comment:
how about having npm run build right next to deploy and sandbox start separately? the sandbox start is run just once,  however each time you make a change to the  ts file you need to recompile it. Also, it would be good to write somewhere what is the ts file so that it is encouraged to explore it and change.

Reply:
> how about having npm run build right next to deploy and sandbox start separately? the sandbox start is run just once, however each time you make a change to the ts file you need to recompile it. 

Sure